packages/charts-components/templates/angular/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "charts-components-angular-template",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "yarn copy && ng serve",
+    "build": "yarn copy && ng build",
+    "watch": "yarn copy && ng build --watch --configuration development",
+    "test": "ng test",
+    "copy": "copy-folder ./node_modules/@arcgis/charts-components-angular ./charts-components-angular"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "^17.1.0",
+    "@angular/common": "^17.1.0",
+    "@angular/compiler": "^17.1.0",
+    "@angular/core": "^17.1.0",
+    "@angular/forms": "^17.1.0",
+    "@angular/platform-browser": "^17.1.0",
+    "@angular/platform-browser-dynamic": "^17.1.0",
+    "@angular/router": "^17.1.0",
+    "@arcgis/charts-components-angular": "^4.29.0-beta.41",
+    "@arcgis/charts-model": "^4.29.0-beta.41",
+    "@arcgis/core": "^4.28.10",
```

Comment:
for all these dep updates, they will be managed via syncpack when we open the release branch for 4.29. It's safe to let syncpack manage them via `fix-mismatches` for now. It's not required to manually change them for this PR

Reply:
> It's safe to let syncpack manage them via fix-mismatches for now

Trust but verify since a lot of us are still learning, 😄. This is probably a separate discussion/learning curve for after release. 

In this PR, I want to call out the 4.29-beta vs 4.28 production version mismatches across different libraries, where everything should be `4.29*`. Also want to ensure the @arcgis/core version qualifier is correctly set with syncpack. It should be a tilda `~` and not a caret `^`. 

I expected to see a github action bot creating a commit against this PR so those updates could be reviewed "inline" and tested locally before the 4.29 release branch is opened. I wouldn't want to wait before testing, or maybe I'm misunderstanding how all the syncpack pieces fit together. I'll move this discussion offline.

Reply:
> In this PR, I want to call out the 4.29-beta vs 4.28 production version mismatches across different libraries, where everything should be 4.29*. Also want to ensure the @arcgis/core version qualifier is correctly set with syncpack. It should be a tilda ~ and not a caret ^.

Yep, running `npx syncpack fix` will put all these versions at `next` / `rc` (`rc` for core because next is already 4.30). that's controlled here: https://github.com/Esri/arcgis-maps-sdk-javascript-samples-beta/blob/a755d8c29d224348ba55a98ff04b77e4f3ff2c27/.syncpackrc.js#L50-L70

Once this PR is merged. we'll update that file, run `syncpack fix-mismatches` again which will set ALL the packages in this repo to use `~4.29.0` for core and each component package. Then we'll do one final test on that branch and merge to main. That's how I picture things going.

> I expected to see a github action bot creating a commit against this PR so those updates could be reviewed "inline" and tested locally before the 4.29 release branch is opened. I wouldn't want to wait before testing, or maybe I'm misunderstanding how all the syncpack pieces fit together. I'll move this discussion offline.

yeah, no bot yet, but this sounds like a good idea. We should iron out the details in the next couple weeks

Reply:
Cool thanks, that's what I figured. The workflow that was tripping me up was I wanted to test Jimmy's branch locally. Wasn't sure how to reconcile the deps I was testing with the 4.29 branch syncpack-switcheroo.